features/mbedtls/targets/TARGET_STM/TARGET_STM32F4/aes_alt.c

```diff
@@ -0,0 +1,364 @@
+/*
+ *  Hardware aes implementation for STM32F4 STM32F7 and STM32L4 families
```

Comment:
Would a newer version of this `aes_alt.c` file work for both `TARGET_STM32F4` and `TARGET_STM32L4`? It seems silly to have two different nearly identical copies of the same file in different places in the Mbed OS tree. I'd prefer we use ifdefs to handle the minor differences between these targets as needed. Same goes for other files added by this PR that are nearly the same as existing files.

Also, please ensure it is very clear which version of the ST SDK these files are from, for easier maintenance.